CMakeLists.txt

```diff
@@ -55,6 +58,6 @@ target_link_options(kernel PRIVATE -T linker.ld -ffreestanding -O2 -nostdlib -no
 
 add_custom_command(
     TARGET kernel
-#    POST_BUILD
-#    COMMAND bash post.sh
+    POST_BUILD
+    COMMAND bash post.sh

```

Comment:
This was removed for a reason.
This needs to be a discrete step in the build process

Reply:
In that case we should remove it completly. The way it ws commneted out gave cmke errors stopping the generation.